steelthread-0/walkthrough.md

```diff
@@ -25,62 +28,68 @@ You should also take the opportunity to login your oc CLI as this user. You can
 - Collapse the provisioned service and click on the dashboard link which will show on the right hand side of the EnMasse provisioned service. This link will bring you to the EnMasse dashboard. 
 - When asked to login click the OpenShift button and login with the same user that provisioned the service. In an eval environment this will be ```evals@example.com```
 - Create a new address in the EnMasse console named ```work-queue/requests``` choose the ```queue``` option and the pooled plan.
-- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the multicast option and the standard plan
+- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the ```queue``` option and the standard plan
 
-# Setup Enmasse connection in Fuse Ignite
+# Deploy the CRUD REST HTTP booster
 
-Back in your ```evals``` project, You will see a binding and a binding secret as part of the Enmaase provisioned service. There is an option to view this secret. Click on view and click on reveal. Take note of the following values
+This is the HTTP endpoint that will revieve the requests from the Fuse Ignite 
+integreation we create and add new fruits to the inventory.
 
-- messagingHost
-- username
-- password
 
-open the Fuse Ignite route and login in to the Fuse Ignite console using the evals@example.com user 
+Load up the launcher application. This is at ```http://launcher-launcher.<BASE_MASTER_DOMAIN> ``` 
 
-- Select connections and create connection
-- Find the AMQP connection and select it. 
-- Fill in the Connection URI Field with the a URI that is in the following format  ```amqp://<messagingHost>:5672?amqp.saslMechanisms=PLAIN```
-- Fill in the username and password fields
-- Disable Check Certificates
-- Click Validate 
-- Click the blue ```next``` button at the top right
-- Add a connection name ```enmaasse```
-- Click Next / Save
+- Click on launch your project
+- Login when prompted using the ```evals@example.com``` user
+- The create application form will show. Choose ```eval``` as the name (this equates to your namespace)
+- When prompted to select target environment choose : ```Code Locally, Build and Deploy```
+- Click the blue down arrows
+- Under mission select ```Crud```
+- Under runtime select ```Node.js```
+- Click the down arrows
+- Under ``` Authorize Git Provider``` you will need to authorize launcher to have access to your github account.
+- Once authorised, under the repository input change the value to ```crud-app```
+- Once the down arrows go blue click them
+- Click setup application.
+- This will create a repo called crud-app in your github account and deploy the CRUD booster to the eval namespace.
+
+You can try out this application by going to the eval namepsace and clicking on the route created for this new application.
 
 
 # Deploy the messaging booster
+**TODO at the moment we do this manually in the future it will be done via launcher**
```

Comment:
This is no longer the case. We can now provision the messaging frontend via launcher.

Messaging Booster -> Node.js -> Integreatly

Reply:
Updating this section to use launcher